src/hipscat/io/validation.py

```diff
@@ -59,16 +65,16 @@ def handle_error(msg):
         is_valid = False
 
     if not is_catalog_info_valid(pointer, storage_options=storage_options):
-        handle_error("catalog_info.json file is invalid.")
+        handle_error("catalog_info.json file does not exist.")
```

Comment:
We're doing some catalog info validation in this one, maybe we want to keep the message as is.